### ISSUE TITLE
[$10][MSFT-22] [Challenge Feed] TCO icon contrast between foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2745]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. In 'My Challenges' list; check the TCO icon (eg: Cd, F2F, Wb, DF2F, FE, Wg, MM)

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #2C4C0F. New contrast ratio: 4.5
- Passes AA, if you change text color to #3E8000. New contrast ratio: 4.54

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 2.01 (foreground color: #60c700, background color: #effae5, font size: 7.5pt, font weight: bold).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=false&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F7

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096158 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096158 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
@Dara-K can you please suggest the color for this one? Thanks.

**Comment by Dara-K** [comment]
@r0hit-gupta I have updated the color pallette we use for TC in particular the dark shades of color: https://marvelapp.com/c7h2a6h/screen/59648056/handoff 

For your question: 
On development: green shape with Cd and TCO text should be # 258205
On design: blue shape and TCO text should be # 0076A5
On MM: orange shape and TCO text should be # BA4C00

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2745 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2745 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2745 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2745 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097467 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097467 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
> On design: blue shape and TCO text should be # 0076A5

@Dara-K I can't find #0076A5 on color palette.

**Comment by tosha5252** [comment]
Text and shape color for the icons are not changed to what DaraK has in the comment

Green shape and text color is  # 5cc900
Blue Shape and color is  # 15acec
Orange shape and color is #f d7d00

Please change to:
On development: green shape with Cd and TCO text should be # 258205
On design: blue shape and TCO text should be # 0076A5
On MM: orange shape and TCO text should be # BA4C00

Env:
Windows 10
Browser: Chrome

**Comment by r0hit-gupta** [comment]
@tosha5252 please check on https://test-community-app.topcoder-dev.com/challenges

It is showing the correct colors. You might need to run `npm install` again as the `react-ui-kit` was updated with the new colors. Also, I am using `#006ad7` for the blue shape as I could not find the color `#0076A5` in the palette https://marvelapp.com/c7h2a6h/screen/59648056/handoff . All the colors are passing WCAG AA requirement.

**Comment by lakshmiathreya** [comment]
@r0hit-gupta could you clarify the final decision on all 3 colors pls so we have a consistent reference ... fyi @tosha5252 @drasticdpk

**Comment by r0hit-gupta** [comment]
> @r0hit-gupta I have updated the color pallette we use for TC in particular the dark shades of color: https://marvelapp.com/c7h2a6h/screen/59648056/handoff
> 
> For your question:
> On development: green shape with Cd and TCO text should be # 258205
> On design: blue shape and TCO text should be # 0076A5
> On MM: orange shape and TCO text should be # BA4C00

@lakshmiathreya @tosha5252 as per @Dara-K original comment, all the colors were specified in the palette link except the blue color `#0076A5`. I think this might be a typo, so used one of the blue colors from the given palette link which passed the WCAG AA requirement. I hope this clears it up. 

@Dara-K Please confirm once again.

**Comment by crazyk07** [comment]
@r0hit-gupta  Think you made a typo on "green" .  On dev I see #258025, but it should actually be 258205.  Probably a Typo.  

The others I see as accurate. @lakshmiathreya @tosha5252 

Chrome
Windows 10

**Comment by crazyk07** [comment]
![image](https://user-images.githubusercontent.com/43656195/62145469-64fc0a80-b2a8-11e9-89ab-d1a0c169d0bd.png)

**Comment by r0hit-gupta** [comment]
@crazyk07 all colors are being used from https://marvelapp.com/c7h2a6h/screen/59648056/handoff . Please let me know if this palette needs to be updated.

**Comment by Dara-K** [comment]
Yes, please use #006ad7 for blue and for green is #258205

**Comment by r0hit-gupta** [comment]
@crazyk07 @lakshmiathreya new PRs - https://github.com/topcoder-platform/community-app/pull/3165 & https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/21 should solve this now. Please review. 

Also, please note that PR https://github.com/topcoder-platform/community-app/pull/3165 depends upon https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/21

**Comment by lakshmiathreya** [comment]
Green and Orange shapes seem accurate - 258025 and BA4C00 resp.
But Blue shape is 006ad7 - not 0076A5. Pls clarify @r0hit-gupta @crazyk07. fyi @tosha5252 @drasticdpk

**Comment by r0hit-gupta** [comment]
> @r0hit-gupta I have updated the color pallette we use for TC in particular the dark shades of color: https://marvelapp.com/c7h2a6h/screen/59648056/handoff
> 
> For your question:
> On development: green shape with Cd and TCO text should be # 258205
> On design: blue shape and TCO text should be # 0076A5
> On MM: orange shape and TCO text should be # BA4C00

@lakshmiathreya check this out. https://github.com/topcoder-platform/community-app/issues/2745#issuecomment-514807435

**Comment by Dara-K** [comment]
006AD7 is the color for the links and buttons - the main blue color
0076A5 is the color for the blue for Design track.
https://marvelapp.com/c7h2a6h/screen/59648056/handoff

**Comment by sushilshinde** [comment]
kit needs deployment

**Comment by tosha5252** [comment]
verified changed

Windows 10 env
Browser: Chrome

**Comment by tosha5252** [comment]
Verified

Win10/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097467<br/><br/>```This is an automated message for crazyk via Topcoder X```
